### PR TITLE
Added Pos/SetPos methods to Window

### PR DESCRIPTION
**Pull request by Drahoslav7**
Hi, I added support to change position of window. Might be relative to #145 .

Should there also be possibility to set position at CreateWindow?

### CODE REVIEW

File: viewport.go [hunk 1]
```diff
@@ -40,6 +40,13 @@ type Viewport interface {
 	// SetTitle changes the title of the window.
 	SetTitle(string)
 
+	// Pos returns position of the window.
+	Pos() (int, int)
+
+	// SetPos changes position of the window.
+	SetPos(int, int)
+
+
```
Comment (shurcooL) [inline viewport.go]: This isn't gofmted.

**Comment by ben-clayton** [comment]
Hi @Drahoslav7,

Thanks for the PR. I have two minor comments.
It would be more consistent with the rest of GXUI if:
* The position was represented as a [math.Point](https://github.com/Google/gxui/blob/master/math/point.go#L7) instead of two ints.
* `Position` was used instead of `Pos`

However, the logic looks good.

Many thanks,
Ben

**Comment by Drahoslav7** [comment]
Thanks for reviewing @ben-clayton,
I commited some changes according to your commets.

**Comment by ben-clayton** [comment]
LGTM - Thank you!

**Pull request merged**